app/views/foods/index.html.erb

```diff
@@ -0,0 +1 @@
+<h2>Number of items: <%= @foods.count %></h2>
```

Comment:
This is a great start. We will want to iterate over foods here to get our list and all their attributes.